### PR TITLE
feat: package up project with a zip file (for mac os)

### PR DESCRIPTION
**Pull request by kaushikgopal**

### CODE REVIEW

File: package_up [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+zip_filename=instacart-pp-$(whoami).zip
+pwd=$(pwd)
+
+
+echo "-- cleaning build files"
+rm -rf build
```
Comment (colinrtwhite) [inline package_up]: The are build files in the module folder too, right? Maybe we should run `./gradlew clean` instead?

Comment (kaushikgopal) [reply package_up]: i suppose this is why they say code reviews are a good thing :P. 🙏

**Comment by kaushikgopal** [comment]
fwiw: this only handles mac os-es. linux and windows would probably need something slightly more sophisticated.

File: package_up [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+zip_filename=instacart-pp-$(whoami).zip
```
Comment (FrancoisBlavoet) [inline package_up]: 👍 on the `whoami` .. 
I have been adding bonus points for candidates adding their names to the archive

I have just tested this command with my last candidate, it worked like a charm !

**Pull request merged**